setup.py

```diff
@@ -138,15 +138,20 @@ class BuildFailed(Exception):
     def __init__(self, ext):
         self.ext = ext
 
+is_windows = sys.platform.startswith('win32')
+is_macos = sys.platform.startswith('darwin')
 
 murmur3_ext = Extension('cassandra.cmurmur3',
                         sources=['cassandra/cmurmur3.c'])
 
+libev_includes = ['/usr/include/libev', '/usr/local/include', '/opt/local/include', '/usr/include']
+if is_macos:
+    libev_includes.extend(['/opt/homebrew/include', os.path.expanduser('~/homebrew/include')])
```

Comment:
Just making the [recent changes](https://github.com/datastax/python-driver/commit/2c61ab22792475445d21b77bd165cf330ee9b87b) to support builds using Homebrew constrained to OSX builds only